### PR TITLE
Set active tab of layer settings

### PR DESCRIPTION
**Pull request by vlt1**
## Description
Added initialActiveTab cfg prop to TOCItemsSettings plugin.

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [x] Other... Please describe: enhancement

## Issue
#5427

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/components/TOC/enhancers/tocItemsSettings.js [hunk 1]
```diff
@@ -101,6 +101,7 @@ const settingsLifecycle = compose(
         },
         componentWillUpdate(newProps) {
             const {
+                initialActiveTab = 'general',
```
[the remark this answers is not in the document]
Comment (vlt1) [reply web/client/components/TOC/enhancers/tocItemsSettings.js:104]: @MV88 `npm run doctest` fails even on master.

[the remark this answers is not in the document]
Comment (offtherailz) [reply web/client/components/TOC/enhancers/tocItemsSettings.js:104]: I had the same problem. 
I removed docma all acorn/jsdoc depedencies (even custom) from node_modules and re-install them and then I was able to build again. It worked for me. Probably there was some update in libs to support conditional chaining.

[the remark this answers is not in the document]
Comment (MV88) [reply web/client/components/TOC/enhancers/tocItemsSettings.js:104]: has this been solved?

[the remark this answers is not in the document]
Comment (vlt1) [reply web/client/components/TOC/enhancers/tocItemsSettings.js:104]: @MV88 yes, what @offtherailz said solved the problem for me

**Pull request merged**